ingest/src/main/scala/hydra.ingest/http/SchemasEndpoint.scala

```diff
@@ -92,13 +92,11 @@ class SchemasEndpoint()(implicit system: ActorSystem)
   }
 
   private val v2Route =
-    pathPrefix("v2" / "topics") {
-      ignoreTrailingSlash {
-        pathSuffix("schema") {
-          get {
-            extractUnmatchedPath { subject =>
-              getSchema(includeKeySchema = true, subject.toString().replace("/", ""), None)
-            }
+    pathPrefix("v2") {
+      get {
+        path("schemas" / Segment) { subject =>
+          pathEndOrSingleSlash {
```

Comment:
👍 I like `pathEndOrSingleSlash` better than `ignoreTrailingSlash` because `ignoreTrailingSlash` can cause the route to get processed twice which can be an issue in some cases (see https://doc.akka.io/docs/akka-http/current/routing-dsl/directives/path-directives/ignoreTrailingSlash.html)

Reply:
FYI: If you do a "pathEndOrSingleSlash" the path preceding cannot be just "path" if Segment is the last piece of that route (rejects a single slash at end of route). If you use "pathPrefix" however, the route does allow the single slash at the end. Confusing...I know.